### PR TITLE
multistorage -  map storageclass to flashSystemCluster

### PR DESCRIPTION
**Pull request by bvered**
multistorage feature  - map storageClass to FlashSystemCluster using secret

### CODE REVIEW

**Comment by bvered** [comment]
[Multi storage- add storageClass to storageSystem configmap mapping by comparing secret host address](https://trello.com/c/qx3QHl7Z/91-multi-storage-add-storageclass-to-storagesystem-configmap-mapping-by-comparing-secret-host-address)

File: controllers/storageclass/storageclass_controller.go [hunk 1]
```diff
@@ -116,6 +119,11 @@ func (r *StorageClassWatcher) Reconcile(_ context.Context, request reconcile.Req
 		return result, err
 	}
 
+	flashSystemCluster, err := r.getFlashSystemClusterByStorageClass(sc)
+	if err != nil {
```
Comment (tyichye19) [inline controllers/storageclass/storageclass_controller.go:123]: if err != nil then we have an error and we need to return. maybe you mean err == nil?

File: controllers/util/poolutil.go [hunk 2]
```diff
@@ -30,6 +30,10 @@ const (
 
 	CsiIBMBlockDriver = "block.csi.ibm.com"
 	CsiIBMBlockScPool = "pool"
+
+	SecretName              = "csi.storage.k8s.io/provisioner-secret-name"
+	SecretNamespace         = "csi.storage.k8s.io/provisioner-secret-namespace"
```
Comment (tyichye19) [inline controllers/util/poolutil.go:35]: maybe `SecretNameKey` /`SecretNamespaceKey` would be more accurate?

File: controllers/storageclass/storageclass_controller.go (hunk 1 above)
Comment (bvered) [reply controllers/storageclass/storageclass_controller.go]: fixed

File: controllers/util/poolutil.go (hunk 2 above)
Comment (bvered) [reply controllers/util/poolutil.go]: fixed

**Review by tyichye19 — APPROVED** [review]
LGTM

**Pull request merged**